src/_metronic/layout/components/sidebar/sidebar-menu/SidebarMenuMain.tsx

```diff
@@ -7,9 +7,10 @@ import {useProtectedRoute} from '../../../../../app/components/RouteProtection/P
 
 const SidebarMenuMain = () => {
   const intl = useIntl()
-  const allRolesRoutes = useProtectedRoute(['Admin', 'Scrutiny'])
+  const allRolesRoutes = useProtectedRoute(['Admin', 'Scrutiny', 'Promoter'])
```

Comment:
it is good create a constant that contains roles string, and create a new constants creating a array with constants of roles